### ISSUE TITLE
Is there a way to create fixed height container?

### ISSUE THREAD

**Issue by Vamp808** [QuestPDF/QuestPDF#828]
Lets say we have a container "ex. cell" that has specific dimension, and we want to put a text that is bigger than the cell dimension , 
normally, putting the bigger text inside cell will make the cell height to expand until it contains all the text,
in (iTextSharp) if I set this cell height as (Fixed Height), only part of the text (that the cell height can handle) will show,  the remaining text will not show , and the cell height will not expand , see the image:

![img](https://github.com/QuestPDF/QuestPDF/assets/17684361/b139511a-5c87-41ce-9cf6-0ef5c78b14f0)

I tried (MaxHeight) without success, it throws exception.

so how can I achieve this in QuestPdf?

**Comment by MercinaM** [comment]
You should be able to achieve what you want by using a combination of `MaxHeight` and `StopPaging`. You may also need `WrapAnywhere` for results to show the last partial word ("hav"), but that will only work correctly if are rendering a single line of text; with 2 lines or more, I believe `WrapAnywhere` will cause words to break in lines 1 and 2 as well, which is probably not something you want.

There is a discussion around this topic in [https://github.com/QuestPDF/QuestPDF/discussions/742](url), and from what I gather future versions (after 2024.3.0) may offer better support for text truncation.

**Comment by MarcinZiabek** [comment]
Indeed, the `2024.3.0-rc2` release allows you to set the maximum number of lines that should be visible. If text requires more space, it is truncated with the ellipsis symbol.

**Comment by MarcinZiabek** [comment]
I’m happy to share that QuestPDF now supports limiting the number of visible lines in a paragraph, truncating overflow text with an ellipsis. More details can be found here: [Clamp Line with Ellipsis](https://www.questpdf.com/api-reference/text/paragraph-style.html#clamp-line-with-ellipsis).

If your goal is to discard text content strictly based on cell height, the Stop Paging API provides a suitable solution. For further details, please refer to the documentation: [Stop Paging API](https://www.questpdf.com/api-reference/stop-paging.html).

As there has been no further activity, I’m closing this issue. If you have a similar request in the future, please feel free to open a new GitHub issue. I truly appreciate your feedback! ❤️
